### PR TITLE
Add project name to job info

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

**Comment by fviale** [comment]
Retest this please

File: rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java [hunk 1]
```diff
@@ -228,8 +238,8 @@ public String toString() {
                ", numberOfRunningTasks=" + numberOfRunningTasks + ", numberOfFinishedTasks=" + numberOfFinishedTasks +
                ", numberOfFailedTasks=" + numberOfFailedTasks + ", numberOfFaultyTasks=" + numberOfFaultyTasks +
                ", numberOfInErrorTasks=" + numberOfInErrorTasks + ", priority=" + priority + ", jobOwner='" + jobOwner +
-               '\'' + ", toBeRemoved=" + toBeRemoved + ", genericInformation=" + genericInformation + ", variables=" +
-               variables + '}';
+               "', projectName='" + projectName + "', toBeRemoved=" + toBeRemoved + ", genericInformation=" +
```
Comment (mboussaa) [inline rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java]: ', projectName=' => , projectName=
', toBeRemoved => , toBeRemoved

File: scheduler/scheduler-server/src/test/java/functionaltests/api/TestLoadJobs.java [hunk 2]
```diff
@@ -132,6 +132,7 @@ public void testLoadJobs() throws Exception {
         job = jobs.get(0);
 
         assertEquals(this.getClass().getSimpleName(), job.getJobId().getReadableName());
+        assertEquals(this.getClass().getSimpleName(), job.getProjectName());
```
Comment (mklkun) [inline scheduler/scheduler-server/src/test/java/functionaltests/api/TestLoadJobs.java]: This means that `job.getJobId().getReadableName()` has the same value as `job.getProjectName()` in this test?

File: rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java (hunk 1 above)
Comment (fviale) [reply rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java]: project name can contain spaces, this is why we put it inside quotes, toBeRemoved does not contain spaces

**Pull request merged**